react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ * TODO: The buttons don't work!
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+
+  function handleClickPrev() {
+    setCurrent((current - 1 + items.length) % items.length);
+  }
+
+  function handleClickNext() {
+    setCurrent((current + 1) % items.length);
+  }
+
+  function handleClick(index) {
+    setCurrent(index);
+  }
+
+  return (
+    <div>
+      <div>{items[current]}</div>
+      <div>
+        <CustomButton text="Prev" onClick={handleClickPrev} />
+        <Indicators count={items.length} onClick={handleClick} />
```

Comment:
You're not passing `current` here. This is why the indicators aren't showing the current item.